spec/models/profile_spec.rb

```diff
@@ -39,7 +39,7 @@
         expect(profile.from_omniauth_hash(@from_omniauth)['bio']).to eq('this is my bio')
       end
 
-      it 'does not overwrite any exsisting profile fields' do
+      it 'does not overwrite any existing profile fields' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.